lib/db/schema/patch-034-035.sql

```diff
@@ -0,0 +1,87 @@
+CREATE TABLE IF NOT EXISTS unblockCodes (
+  uid BINARY(16) NOT NULL,
+  unblockCode VARCHAR(32) NOT NULL,
+  createdAt BIGINT SIGNED NOT NULL,
+  UNIQUE INDEX unblockCodes_uid_code(uid, unblockCode)
```

Comment:
We could make (uid, unblockCode) a composite primary key and avoid the need for a secondary index here.  IIUC we'll only ever look up records by the (uid, unblockCode) pair.